examples/app-settings/src/settings.ts

```diff
@@ -23,12 +23,10 @@ export const settings = defineSettings({
             type: 'input',
             id: 'single-line',
             label: 'Input field',
-            defaultValue: 'Input Field Value',
```

Comment:
Default values can sometimes be a bit tricky when using the dev env. I would like to remove them just to prevent errors.